bikeshed/include/html/header.include

```diff
@@ -2,13 +2,11 @@
 <html lang="en">
 <head>
   <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-  <title>[TITLE] [LEVEL]</title>
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
   <meta name="w3c-status" content="[STATUS]">
-  <link href="../default.css" rel=stylesheet type="text/css">
-  <link href="../csslogo.ico" rel="shortcut icon" type="image/x-icon">
-  <link href="styles/styles-html.css" rel=stylesheet type="text/css">
-  <link href="[W3C-STYLESHEET-URL]" rel=stylesheet type="text/css">
+  <title>[TITLE] [LEVEL]</title>
```

Comment:
The title metadata for the document is "HTML" and the level is "5.1", However the &lt;title> and &lt;> is the combined title metadata and the level metadata because each of these metadata items is not completely accurate for the &lt;title> or the &lt;h1>. Additional places in the document use the [TITLE] and [LEVEL] metatdata items separately and not necessarily placed together. Making the title metadata "HTML 5.1" would be incorrect in those cases and all the cases since the title is really "HTML".